docs/README.md

```diff
@@ -29,7 +30,7 @@ new documentation:
 - [Present tense](https://developers.google.com/style/tense)
 
 The guidelines include much more material and are used as a guide and enable
```

Comment:
There are a lot of ands in this sentence.

Reply:
Original content that I didn't dare change. I'll try to de-and it a bit.

Reply:
Fixed, after a fashion